### PR TITLE
[Goodreads] lookup improvement

### PR DESCRIPTION
**Pull request by robert00s**
+ add navigation button (next, prev) on the bottom of screen
+ flash item when selecting
+ small lookup improvements
+ fix refresh "Please wait..."

### CODE REVIEW

**Comment by robert00s** [comment]
screenshot 2017-10-14 12-55-19

File: plugins/goodreads.koplugin/doublekeyvaluepage.lua [hunk 1]
```diff
@@ -116,44 +120,37 @@ function DoubleKeyValueItem:init()
     end
     local key_w = RenderText:sizeUtf8Text(0, self.width, self.cface_down, self.key).x
     local value_w = RenderText:sizeUtf8Text(0, self.width, self.cface_up, self.value).x
-    if key_w > self.width then
-        self.show_key = RenderText:truncateTextByWidth(self.key, self.cface_down, self.width)
+    if key_w > self.width - padding then
```
Comment (Frenzie) [inline plugins/goodreads.koplugin/doublekeyvaluepage.lua]: Is that `-padding` or `-2*padding`?

Comment (robert00s) [reply plugins/goodreads.koplugin/doublekeyvaluepage.lua]: Right, should be 2*padding.

File: plugins/goodreads.koplugin/doublekeyvaluepage.lua [hunk 2]
```diff
@@ -116,44 +120,37 @@ function DoubleKeyValueItem:init()
...
+        width = self.width,
+        height = self.height,
         VerticalGroup:new{
-            dimen = Geom:new{ h = h, w = w },
-            padding = Screen:scaleBySize(10),
             LeftContainer:new{
-                padding = Screen:scaleBySize(10),
-                dimen = Geom:new{ h = h, w = w },
+                padding = 0,
+                dimen = Geom:new{ h = h   , w = w },
```
Comment (Frenzie) [inline plugins/goodreads.koplugin/doublekeyvaluepage.lua]: Btw, why'd you change this from what it was? (Space-wise) Same a few lines down.

File: plugins/goodreads.koplugin/doublekeyvaluepage.lua [hunk 3]
```diff
@@ -231,24 +283,31 @@ function DoubleKeyValuePage:init()
...
+    self[1] = FrameContainer:new{
         height = self.dimen.h,
         padding = padding,
         bordersize = 0,
         background = Blitbuffer.COLOR_WHITE,
-        VerticalGroup:new{
-            self.title_bar,
-            self.main_content,
-        },
+        content
```
Comment (Frenzie) [inline plugins/goodreads.koplugin/doublekeyvaluepage.lua]: Might as well add a comma to possibly reduce the amount of lines changed in the future.

**Pull request merged**